indexer/defaults.py

```diff
@@ -8,7 +8,7 @@
 PATHS_FILE = ""
 BLACKLIST = None
 BLACKLIST_FILE = ""
-URL = "https://file-catalog.icecube.wisc.edu/"
+ADDRESS = "https://file-catalog.icecube.wisc.edu/"
 TIMEOUT = 30  # seconds
 RETRIES = int((60 / TIMEOUT) * _AGGREGATE_LATENCY_MINUTES)
```

Comment:
Again with the high timeout and large number of retries.  Is there a reasoning?

Reply:
https://github.com/WIPACrepo/file-catalog-indexer/pull/60#discussion_r1137610718, I recommend CL ags